R/task.R

```diff
@@ -70,3 +70,31 @@ Task <- R6::R6Class(
     }
   )
 )
+
+#' @title activateWorkflowTasks
+#' @description activates a series of `Tasks` from a `Workflow`
+#' @param workflow `MeanModelWorklfow` or `PopulationWorklfow` object
+#' @param tasks names of the tasks to activate
+#' Default activates all tasks of the workflow using workflow method `workflow$getAllTasks()`
+#' @export
+activateWorkflowTasks <- function(workflow, tasks = workflow$getAllTasks()) {

```

Comment:
Should not that be a method of the workflow ?

Reply:
Yes, 
in case, I kept the methods within and outside of the workflow, so the user can choose both ways.